src/mame/drivers/dgpix.cpp

```diff
@@ -15,6 +15,8 @@
  [*] the version number is written in the flash roms at the beginning of the game settings
 
 Note: There is known to exist an alternate version of The X-Files titled The Sex Files which is undumped
+An additional game called Beat Player 2000 is undumped, using the VRenderO Minus Rev5 PCB,
+with an undocumented BMkey Flash Rev2 dgPIX Entertainment 1999 subboard.
```

Comment:
It’s zero, not oh.  Also, use descriptive PR titles/descriptions and commit messages.  Given this very basic error, I’m not confident in the veracity of the comment.  Can you provide a citation for the information?

Reply:
https://twitter.com/dan_evileye/status/723152981009399808?s=19 and http://blog.system11.org/?p=1396. I'm going to add let's dance (which david already linked), as well as Fishing Maniac http://blog.system11.org/?p=1396 and Fishing Maniac 2 http://blog.system11.org/?p=2182. I'm sorry about the bad PR name and commit message, I confess I've barely used github. I also apologize for the error.

Reply:
Notice the whole driver uses the _VRenderO_ cargo cult naming, and should be bulk renamed.

Reply:
VRenderO Minus, it's important to specify

VRender0 = Crystal System, done in hardware
VRender0 Minus = this, done on a Hyperstone CPU in software

Reply:
fwiw gp2x also uses SoC solutions derivative of the VRender3D design

Reply:
> Notice the whole driver uses the _VRenderO_ cargo cult naming, and should be bulk renamed.

Should I do that here, too? Or leave someone else to do it.

Reply:
I replaced all instances of VRenderO with VRender0 in the driver. This PR should now be finalized.